app/store/sagas/github.js

```diff
@@ -1,39 +1,32 @@
-/**
- * @module Sagas/GitHub
- * @desc GitHub
- */
-
 import {all, call, put, takeLatest} from 'redux-saga/effects';
 
 import gate from 'gate';
-import {ActionTypes} from 'store/constants/index';
-
-/**
- * Get Repos
- *
- * @param {Object} action
- *
- */
-export function* getRepos({payload}) {
+import i18n from 'i18n';
+
+import {
+  getRepos,
+  getReposSuccess,
+  getReposFailure,
+} from 'store/reducers/github';
+
+const {githubLang} = i18n;
+
+function* getReposSaga({payload}) {
   try {
     const response = yield call(gate.getRepositories, payload.query);
 
-    yield put({
-      payload: {data: response.items},
-      type: ActionTypes.GITHUB_GET_REPOS_SUCCESS,
-    });
+    yield put({payload: {data: response.items}, type: getReposSuccess.type});
   } catch (err) {
-    /* istanbul ignore next */
-    yield put({
-      payload: err,
-      type: ActionTypes.GITHUB_GET_REPOS_FAILURE,
-    });
+    let text = githubLang.requestError;
+
+    if (err && err.data && err.data.message) {
```

Comment:
In new `babel` a new amazing syntax is released that name is `optional chaining` which causes you can write line 22 like below:

    err?.data?.message

This new feature needs the `@babel/plugin-proposal-optional-chaining` plugin. If you are agreed add it and change this line.

Reply:
this is done, 
but I couldn't handle `eslint` configuration ([mentioned here](https://github.com/microsoft/vscode-eslint/issues/535))
for VSCode for now I added 
`{"javascript.validate.enable": false}` to `.vscode/settings.json` 
and it's gone

Reply:
We could ignore this ESLint rule until it will be fixed in the future.